constants.bzl

```diff
@@ -2,3 +2,19 @@
 RELEASE_ENGINES_NODE = "^12.20.0 || ^14.15.0 || >=16.10.0"
 RELEASE_ENGINES_NPM = "^6.11.0 || ^7.5.6 || >=8.0.0"
 RELEASE_ENGINES_YARN = ">= 1.13.0"
+
+SNAPSHOT_REPOS = {
+    "@angular/cli": "angular/cli-builds",
+    "@angular/pwa": "angular/angular-pwa-builds",
+    "@angular-devkit/architect": "angular/angular-devkit-architect-builds",
+    "@angular-devkit/architect-cli": "angular/angular-devkit-architect-cli-builds",
+    "@angular-devkit/build-optimizer": "angular/angular-devkit-build-optimizer-builds",
+    "@angular-devkit/build-ng-packagr": "angular/angular-devkit-build-ng-packagr-builds",
```

Comment:
Where is this referenced? This package has long been removed.

Reply:
Good to know, I'll remove this entry.

https://github.com/angular/angular-cli/blob/master/.monorepo.json

Want me to remove that reference in `.monorepo.json` in this PR?

Reply:
Added